docs/examples/video_decode_remap/README.md

```diff
@@ -0,0 +1,83 @@
+# GPU-accelerated video processing in Triton using DALI
+
+This example presents, how to approach GPU-accelerated video processing in Triton using DALI.
+
+### First steps
+
+1. If you do not know yet what DALI is, please refer to [DALI's getting started guide](https://docs.nvidia.com/deeplearning/dali/user-guide/docs/examples/getting_started.html#Overview)
+1. If you are not yet familiar with the idea of using DALI within Triton, please refer to the
+[How to use?](https://github.com/triton-inference-server/dali_backend#how-to-use) guide and
+[Identity example](https://github.com/triton-inference-server/dali_backend/tree/main/docs/examples/identity).
+These two guides explain the concept and are good starting points.
+
+## GPU-accelerated video processing
+
+While in most of data processing libraries (image or video, like PIL, OpenCV etc...), data decoding
+is transparent for user, in DALI we do put emphasis in this step. Actually, whenever `cv2.imread()`
+function is used, or `PIL.Image.open()`, the decoding happens underneath. In DALI, user explicitly
+calls decoding step, thus gaining a opporunity to use accelerate this operation using GPU.
+
```

Comment:
This point is slightly unclear to me. Should it be about decoding being transparent or rather coupled with loading step? Also the second sentence kind of breaks the flow from the second part of the first sentence.

How about?

```suggestion
In most data processing libraries (image or video, like PIL, OpenCV etc...), data decoding is fused with data loading. For example, whenever `cv2.imread()` or `PIL.Image.open()` is used, the decoding happens automatically under the hood. In DALI, user explicitly calls decoding step, gaining greater control over the process, for example to accelerate this operation using GPU.

```

Reply:
Thanks! Now let me make a review remark on what I've proposed. :D
I would replace the word `fused` with `mixed` or `coupled`, as the fusion seems to be a hot topic with different, and most importantly positive, meaning.